src/components/VideoFeed.jsx

```diff
@@ -28,45 +34,39 @@ class VideoFeed extends Component {
   }
 
   filterNewest(filtered) {
+    const { numVideos } = this.props;
     filtered.sort(this.compDate);
-    filtered = filtered.filter((i, index) =>
-      (index < parseInt(this.props.numVideos))
-    );
-    return filtered
+    return filtered.filter((i, index) => (index < numVideos));
   }
 
   filterQuery(videos) {
+    const { owner } = this.props;
```

Comment:
Try to load any profile page
eg: http://localhost:3000/profile/36
--> Failed prop type: Invalid prop owner of type string supplied to VideoFeed, expected number.

You have to type caste the prop:
const { owner } = parseInt(this.props);

Reply:
Or rather change this in the Profile.jsx

  componentDidMount() {
    this.setState({ user: parseInt(this.props.match.params.id) })
  }

Reply:
I will just pass the prop as a number, since it is a number.